### PR TITLE
Better button focus fix

### PR DESCRIPTION
**Pull request by maximusunc**
Add a custom button component that removes its focus after click

### CODE REVIEW

**Comment by maximusunc** [comment]
Fixes #35

File: src/Input.svelte [hunk 1]
```diff
@@ -9,6 +9,10 @@
     let historyIndex = 0;
 
     function handleKeyPress(event) {
+        // don't allow any default keypress actions
+        // this is mainly to disable scrolling down
+        // when pressing the space bar
+        event.preventDefault();
```
Comment (patrickkwang) [inline src/Input.svelte:15]: Does this keep us from typing in the input when it _does_ have focus? Something is...

**Review by patrickkwang — CHANGES_REQUESTED** [review]
This seems to work great _except_ when the input box actually does have focus.

File: src/Input.svelte (hunk 1 above)
Comment (maximusunc) [reply src/Input.svelte]: Good catch

**Review by patrickkwang — APPROVED** [review]

**Pull request merged**